examples/backtrace.rs

```diff
@@ -1,7 +1,17 @@
 extern crate backtrace;
 
-use backtrace::Backtrace;
+use backtrace::{format_trace, Backtrace, PrintFmt};
 
 fn main() {
-    println!("{:?}", Backtrace::new());
+    print!("{:?}", Backtrace::new());
+    format_trace(Utf8(std::io::stdout()), PrintFmt::Short).unwrap();
+    format_trace(Utf8(std::io::stdout()), PrintFmt::Full).unwrap();
+}
+
+struct Utf8<W>(W);
+
+impl<W: std::io::Write> std::fmt::Write for Utf8<W> {
+    fn write_str(&mut self, s: &str) -> Result<(), std::fmt::Error> {
+        self.0.write_all(s.as_bytes()).map_err(|_| std::fmt::Error)
```

Comment:
Mind backing this out by the time this lands?

Useful for context though!

Reply:
Do you mean removing changes to this file entirely? Or should they be a separate example program?

Reply:
Oh just in the sense that this example file was intended to just print one backtrace, but this might make a good `# Examples` section for the new API?